packages/amplify_flutter_web/lib/amplify_flutter_web.dart

```diff
@@ -1,14 +1,17 @@
 import 'dart:async';
 import 'dart:core';
+
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 
 import 'package:amplify_flutter/amplify.dart';
 import 'package:amplify_flutter_web/types/AmplifyJS.dart';
 import 'package:amplify_flutter_web/interop/types.dart';
 import 'package:amplify_core/interop/utilities/object.dart';
 
+/* copy awsmobile from a web project's aws-exports.js file */
+const awsmobile = {};
```

Comment:
@haverchuck do you have any issue with this being brought in? I have this in my branch as well. This was the only way I could get storage working. I think @ragingsquirrel3 had issues configuring analytics without this as well.

Reply:
I believe that the amplify-js categories can be configured with the straight json (which is what you're doing) or with Object keys corresponding to a cateogry name (my old way).  I personally prefer the object keys, but his should work too.